### PR TITLE
Add tokens `<CORRUPTED_AREA_START>` & `<CORRUPTED_AREA_END>`

### PR DESCRIPTION
**Pull request by SagiPolaczek**
Followed the steps:

1. Added to `special_tokens.py`
2. Ran `fuse-drug/fusedrug/data/tokenizer/modulartokenizer/add_multi_tokenizer_special_tokens.py`

### CODE REVIEW

**Review by floccinauc — CHANGES_REQUESTED** [review]
Looks good - just add the new tokens to bmfm_modular_tokenizer and bmfm_extended_modular_tokenizer, and make sure they're mapped to the same indices (they should do that automatically, but check just in case)

File: fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/gene_tokenizer.json [hunk 1]
```diff
@@ -2612,6 +2612,24 @@
       "rstrip": false,
       "normalized": false,
       "special": true
+    },
+    {
+      "id": 290,
```
Comment (floccinauc) [inline fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/gene_tokenizer.json:2617]: You missed one: _END> should be 291

**Review by floccinauc — CHANGES_REQUESTED** [review]
All's good, except for one start-end token switch you missed (marked by comment)

File: fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/gene_tokenizer.json (hunk 1 above)
Comment (SagiPolaczek) [reply fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/gene_tokenizer.json:2617]: Fixed

**Review by floccinauc — APPROVED** [review]
LGTM

**Pull request merged**